modules/data.atmosphere/R/met2cf.module.R

```diff
@@ -0,0 +1,73 @@
+.met2cf.module <- function(raw.id, register, met, dir, machine, site, start_date, end_date, 
+                           con, host, overwrite=FALSE, format.vars) {
+  logger.info("Begin change to CF Standards")
+  
+  input.id  <-  raw.id$input.id[1]
+  pkg       <- "PEcAn.data.atmosphere"
+  formatname <- 'CF Meteorology'
+  mimetype <- 'application/x-netcdf'
+  format.id <- 33
+  
+  # Probably are other formats for which the CF file is not site specific?
+  new.site.id <- ifelse(met %in% c("NARR"), register$siteid, site$id) 
```

Comment:
This is one of those things I don't really know about. Moved the check to met.process but otherwise left alone.